app/assets/javascripts/repositories/repository_datatable.js.erb

```diff
@@ -1165,14 +1165,14 @@ var RepositoryDatatable = (function(global) {
       var column = TABLE.column(li.attr('data-position'));
 
       if (column.visible()) {
-        self.addClass('glyphicon-eye-close');
-        self.removeClass('glyphicon-eye-open');
+        self.addClass('fa-eye-slash');
```

Comment:
do we also need to add .fas class beside that or no?

Reply:
No, fas is defined in the html.